app/docs/md/enhance-styles/cribsheet.md

```diff
@@ -2,7 +2,35 @@
 title: Cribsheet
 ---
 
-Every Enhance app ships with a preconfigured, dynamic cribsheet to help you find the right Enhance Styles class for the job.
+Most Enhance apps ship with a preconfigured, dynamic cribsheet to help you find the right Enhance Styles class for the job.
 
 Just start your project (`npm start`) and navigate to `/_styleguide/cribsheet`. Here you'll be able to search through all the utility classes and custom properties available to your app. Any [customizations](/docs/enhance-styles/customization) you make to your styleguide will automatically be reflected in your cribsheet.
 
+If you are [migrating from an Architect project](https://enhance.dev/cookbook/migrate-from-architect) to an Enhance project or you are starting with our [minimal template](https://github.com/enhance-dev/enhance-starter-project-minimal) then you will need to add the cribsheet to your project.
+
+## Install Dependencies
+
+First install the `@enhance/styles-cribsheet` package
+
+```bash
+npm install @enhance/styles-cribsheet
```

Comment:
I think it's also required that users install Enhance Styles as well, as it won't be included in the minimal template or an Arc migration (necessarily)?

Reply:
🤦  of course. Which leads me to the question, do we even need this update?

Reply:
Hmmm… I guess maybe not? 😂